src/content/docs/apm/apm-ui-pages/monitoring/transactions-page-find-specific-performance-problems.mdx

```diff
@@ -48,9 +48,9 @@ The <DoNotTranslate>**Transactions**</DoNotTranslate> page also may include link
 
 ## Wall-clock time [#wall_clock_time]
 
-Wall-clock time is the amount of time recorded by the clock. New Relic uses wall-clock time for all the transactions and then sums that value across all of the transactions.
+Wall-clock time measures "real-time elapsed" during a specific transaction. For example, you're using a program to monitor transaction speeds. According to the program, the transaction took 30 seconds to complete, but one minute passed in the real world or on the wall clock due to discrepancies across the program. New Relic uses wall-clock time for all the transactions and then sums that value across all of the transactions.
```

Comment:
> According to the program, the transaction took 30 seconds to complete, but one minute passed in the real world or on the wall clock due to discrepancies across the program

Wall clock time has never made sense to me, but to be honest I'm not sure if this update clarifies it. I think I'm getting stuck on "due to discrepancies across the program". What kind of discrepancies, and why? Why would something take 30 seconds according to my program, but a minute in "the real world"? 

Perhaps the challenge here is the focus on the wall-clock? Is the actual issue that people are using _programs that aren't accurate to real-world performance_? 

If so, perhaps we can reframe to be more explicit about that?

Reply:
One other thought—I think the round numbers here add a bit of confusion, particularly with the neat multiplying by two. It creates a sort of impression in my brain that `wall clock time = 2x other time`.

Perhaps do it with two unlike, non-round numbers? Like 22 seconds and 47 seconds.

Reply:
Fixed this!